docs/UserGuide.adoc

```diff
@@ -328,6 +328,44 @@ There is no need to synchronize manually.
 Stops synchronizing of contact information with any previously configured cloud accounts. +
 Format: `unsync`
 
+== Facebook Integration [Coming in v2.0]:
+
+=== Connecting to Facebook: `connectfacebook`
+
+Synchronizes your AddressBook with your personal Facebook account. +
+Format: `connectfacebook`
+
+****
+* Launches a Facebook authorization page in the browser
+****
+
+=== Adding all Facebook Friends: `addallfacebookfriends`
+
+Adds all your Facebook friends into your AddressBook. +
+Format: `addallfacebookfriends`
+
+****
+* First ensures that Facebook has already been connected
+* Adds all the friends to AddressBook
+****
+
+
+=== Adding a Facebook contact: `addfacebookcontact`
+
+Searches for a specific Facebook User to add to AddressBook. +
+Format: `addfacebookcontact USER_NAME`
+
+****
+* First ensures that Facebook has already been connected
+* Searches Facebook database for User
+* Prompts user to choose which contact to add
+* Adds that contact to AddressBook
+****
```

Comment:
Seems to be implementation details that would look great in the developer guide! Not sure if it's relevant to the user though.